DESCRIPTION

```diff
@@ -54,15 +55,18 @@ Suggests:
     nnet,
     pracma,
     ranger,
+    rgenoud,
     rmarkdown,
     testthat,
     xgboost
 RdMacros:
     mlr3misc
-Remotes: 
-    mlr-org/mlr3,
-    mlr-org/mlr3proba
+Remotes:
+    mlr-org/mlr3@leanify_with_misc,
+    mlr-org/mlr3proba,
+    mlr-org/paradox@leanify,
```

Comment:
Can we directly refer to the master branches of all here instead of having to adjust this in one day again?
This would probably require merging all leanify branches in all repos at the same time?